Code/Framework/AzFramework/AzFramework/DocumentPropertyEditor/Reflection/LegacyReflectionBridge.cpp

```diff
@@ -1036,11 +1036,29 @@ namespace AZ::Reflection
                 {
                     const StackEntry& parentNode = m_stack[m_stack.size() - 2];
                     AZ::Serialize::IDataContainer* dataContainer = parentNode.m_classData ? parentNode.m_classData->m_container : nullptr;
-                    // Only add a label numeric label for sequence containers
-                    if (dataContainer && dataContainer->IsSequenceContainer())
+                    if (dataContainer)
                     {
-                        labelAttributeBuffer = AZStd::fixed_string<128>::format("[%zu]", parentNode.m_childElementIndex);
-                        return labelAttributeBuffer;
+                        if (auto indexedChildeOverride = Find(
```

Comment:
typo in indexedChildeOverride
```suggestion
                        if (auto indexedChildOverride = Find(
```

Reply:
fixed